configure.ac

```diff
@@ -5,7 +5,7 @@
 ## Please see the COPYING and CONTRIBUTORS files for details.
 ##
 
-AC_INIT([Squid Web Proxy],[6.1-VCS],[http://bugs.squid-cache.org/],[squid])
+AC_INIT([Squid Web Proxy],[6.2-VCS],[https://bugs.squid-cache.org/],[squid])
```

Comment:
Should not the URI scheme change land in master first?

Reply:
Ideally yes, though its a bit late to get it in this release. I have a followup coming shortly for this and another out-of-sync bit.